### DISCUSSION TITLE
file-saver-es

### DISCUSSION THREAD

**Discussion by cbejensen** [DefinitelyTyped/DefinitelyTyped#57114] [Request a new @types package]
I am using `file-saver-es` instead of `file-saver`. There are already typings for `file-saver`, but even with `@types/file-saver` installed, when I try to import  `file-saver-es` with strict mode on I get an error ("Could not find a declaration file...").

I have tried adding the following configuration options but the error remains:

```jsonc
{
  "compilerOptions": {
    "types": ["file-saver"],
    "types": ["@types/file-saver"], // also tried
    "paths": {
      "file-saver-es": ["file-saver"],
      "file-saver-es": ["@types/file-saver"] // also tried
    }
  }
}
```

I am not sure what the best way to resolve this is - whether I should just change something on my end or something can be updated in this lib - but would appreciate any help. Thank you!

**Comment by BendingBender** [comment]
Made PR for `file-saver-es`: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57172
